vars/neoDeploy.groovy

```diff
@@ -216,11 +237,83 @@ void call(parameters = [:]) {
                 lock("$STEP_NAME:${neoCommandHelper.resourceLock()}") {
                     deploy(script, configuration, neoCommandHelper, configuration.dockerImage, deployMode)
                 }
+                if(configuration.neo.invalidateCache == true && configuration.deployMode == 'mta') {
+                    echo "Triggering invalidation of cache for html5 applications"
+                    invalidateCache(configuration)
+                }
+                else{
+                    echo "Invalidation of cache is ignored. It is performed only for html5 applications and 'invalidateCache' parameter is set to true."
+                }
```

Comment:
My issue here is that the majority of users will not set the invalidateCache flag at all. For them this message is just added noise in the log, which is already very noisy and difficult to understand when searching for errors. My suggestion is to nest the ifs:
```suggestion
                if(configuration.neo.invalidateCache == true) {
                    if (configuration.deployMode == 'mta') {
                        echo "Triggering invalidation of cache for html5 applications"
                        invalidateCache(configuration)
                    } else {
                        echo "Invalidation of cache is ignored. It is performed only for html5 applications."
                    }
                }
```